src/components/NewContact.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+import { useEffect } from "react/cjs/react.production.min";
+import { useFetch } from "../hooks/useFetch";
+import { baseUrl } from "../utils/baseUrl";
+import "./NewContact.css";
+
+const NewContact = ({ contacts, setContacts }) => {
+  console.log("9...", contacts);
+  const [formData, setFormData] = useState({
+    firstName: "",
+    lastName: "",
+    street: "",
+    city: "",
+  });
```

Comment:
Maybe you could create a ```initialState``` variable out of the component – that way, you could also replace it in line 21-24 as well